rbi/core/object.rbi

```diff
@@ -5,4 +5,9 @@ class Object < BasicObject
 
   sig {returns(Integer)}
   def object_id(); end
+
+  sig { params(blk: T.proc.params(arg: T.untyped).returns(T.untyped)).returns(T.untyped) }
+  def yield_self(&blk); end
+
+  alias_method :then, :yield_self
```

Comment:
we're trying to not use aliases in `rbi` as it doesn't work well with IDE.
Could you please write a separate `def` for it?

Reply:
Sure!